tests/types/test_base_types.py

```diff
@@ -459,3 +467,54 @@ class MyLTDD(LooselyTypedDotDict):
     B = MyLTDD(**{
         "a" : None
     })
+
+def test_dotdict_yaml_serialization(capsys):
+    # To enable yaml serialization, we simply annotate the class as a @yaml_object
+    # Note that this annotation be inherited, even though in our case we know that
```

Comment:
?

Reply:
Uhhh, I think I'm missing an important word....thank you. Fixing.

"Note that this annotation be inherited," --> "Note that this annotation cannot be inherited,"